content/id/docs/tasks/tools/kubeadm/high-availability.md

```diff
@@ -0,0 +1,364 @@
+      (nilai semula `:6443`).
+
+    - Tidak direkomendasikan untuk menggunakan alamat IP secara langsung pada _environment cloud_.
+
+    - _Load balancer_ harus dapat berkomunikasi dengan seluruh Node _control plane_ 
+      pada porta yang digunakan apiserver. _Load balancer_ tersebut juga harus mengizinkan trafik masuk pada porta yang didengarkannya.
+
+    - Pastikan alamat _load balancer_ sesuai
+      dengan alamat `ControlPlaneEndpoint` pada kubeadm.
+
+    - Baca panduan [Opsi untuk _Software Load Balancing_](https://github.com/kubernetes/kubeadm/blob/master/id/docs/ha-considerations.md#options-for-software-load-balancing)
+      untuk detail lebih lanjut.
+
+2.  Tambahkan Node _control plane_ pertama pada _load balancer_ dan lakukan pengujian koneksi:
+
+    ```sh
+    nc -v LOAD_BALANCER_IP PORT
+    ```
+
+    - Kegalatan koneksi yang ditolak memang diantisipasi karena apiserver belum
+      berjalan. Namun jika mendapat _timeout_, berarti _load balancer_ tidak dapat berkomunikasi
+      dengan Node _control plane_. Jika terjadi _timeout_, lakukan pengaturan ulang pada _load balancer_ agar dapat berkomunikasi dengan Node _control plane_.
+
+3.  Tambahkan Node _control plane_ lainnya pada grup tujuan _load balancer_.
+
+## Node _control plane_ dand etcd bertumpuk (_stacked_)
+
+### Langkah-langkah untuk Node _control plane_ pertama
+
+1.  Inisialisasi _control plane_:
+
+    ```sh
+    sudo kubeadm init --control-plane-endpoint "LOAD_BALANCER_DNS:LOAD_BALANCER_PORT" --upload-certs
+    ```
+
+    - Kamu bisa menggunakan opsi `--kubernetes-version` untuk mengatur versi Kubernetes yang akan digunakan.
+      Direkomendasikan untuk menggunakan versi kubeadm, kubelet, kubectl, dan Kubernetes yang sama.
+
+    - Opsi `--control-plane-endpoint` harus diatur menuju alamat atau DNS dan porta dari _load balancer_.
+
+    - Opsi `--upload-certs` digunakan untuk mengunggah sertifikat-sertifikat yang harus dibagikan ke seluruh
+      Node _control-plane_ pada klaster. Jika sebaliknya, kamu memilih untuk menyalin sertifikat ke
+      seluruh Node _control-plane_ sendiri atau menggunakan perkakas automasi, silakan hapus opsi ini dan merujuk ke bagian [Distribusi sertifikat manual](#manual-certs) di bawah.
+
+    {{< note >}}
+    Opsi `--config` dan `--certificate-key` pada `kubeadm init` tidak dapat digunakan secara bersamaan, maka dari itu jika kamu ingin menggunakan
+    [konfigurasi kubeadm](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2)
+    kamu harus menambahkan _field_ `certificateKey` pada lokasi pengaturan yang sesuai
+    (berada di bawah `InitConfiguration` dan `JoinConfiguration: controlPlane`).
+    {{< /note >}}
+
+    {{< note >}}
+    Beberapa _plugin_ jaringan CNI membutuhkan pengaturan tambahan, seperti menentukan CIDR IP untuk Pod, meski beberapa lainnya tidak.
+    Lihat [dokumentasi jaringan CNI](/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network).
```

Comment:
this is the correct anchor link https://kubernetes.io/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#jaringan-pod

Reply:
Fixed in bc5c6c5.